src/api.jsx

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const API_URL = 'https://jesjin-nc-news.onrender.com/api';
```

Comment:
You can alter this to make use of baseURL to create an instance, there's an example in the notes for this under axios. It would mean you don't have to use `{}` to insert the URL